CHANGELOG/CHANGELOG-1.11.md

```diff
@@ -1,5 +1,19 @@
 # Changelog 1.11
 
+## 1.11.3 - 2023-07-12
+
+### Changed
+
+- Upgrade Open vSwitch to 2.17.7. ([#5225](https://github.com/antrea-io/antrea/pull/5225), [@antoninbas])
```

Comment:
@tnqn could you help to check and merge the cherry-pick PR https://github.com/antrea-io/antrea/pull/5232, I will trigger all tests after this PR is merged. Thanks.

Reply:
done